pkgs/applications/plasma-mobile/fetch.sh

```diff
@@ -0,0 +1 @@
+WGET_ARGS=( http://download.kde.org/stable/plasma-mobile/21.05 -A '*.tar.xz' )
```

Comment:
I actually had to cheat to get the 21.05 release. Running the fetch script without changing the version downloads 21.05 *and* 21.06 packages here. Odd.

Reply:
That's strange... Here's what I know: `wget` is acting as a (not very smart) link scraper, looking for `<a>` tags with filenames that match the given pattern. This has broken pretty spectacularly in the past when, for example, KDE modified their website to insert most of the page content with JavaScript. I wonder if, at some point, the page source contained links for both versions? It only seems to have 21.05 now, though.

Reply:
I made the same observations at the time. But I did not investigate more deeply. I guessed vaguely it might be following the link to the parent directory, but I don't get why it wouldn't download 21.05 with the 21.06.